gcovr/tests/shadow/reference/coverage.css

```diff
@@ -0,0 +1,429 @@
+:root {
```

Comment:
This file isn't needed in the reference.

Reply:
done